### PR TITLE
Improvements to Content Injection and Data Sync

### PR DESCRIPTION
**Pull request by stubbedev**
Added Injection option for Powerstep in native modal.
Added Category page injection.
Added manual facet attribute addition to admin.
Added support for syncing subscriber status from newsletter module.
Improving product sync performance, reducing size of initial bulk product call.
Added fix for iOS zoom on mobile input fields with font-size under 16px.
Added correct placeholder contents to all options in Admin.
Added toggle visibility for facets button within search page template.

### CODE REVIEW

File: views/templates/hook/search-top.tpl [hunk 1]
```diff
@@ -23,17 +23,6 @@
 * SOFTWARE.
 *}
 
-{*<div id="search_block_top" class="col-sm-4 clearfix">
```
Comment (CasperKN) [inline views/templates/hook/search-top.tpl:26]: Is it on purpose you have comment out this section ?

Comment (stubbedev) [reply views/templates/hook/search-top.tpl:26]: https://github.com/clerkio/clerk-prestashop/blob/master/views/templates/hook/search-top.tpl#L26-L36

was already like this, in my opinion we could just delete it

Comment (CasperKN) [reply views/templates/hook/search-top.tpl:26]: Delete it, and just clean up the files for out commented code, then after that, we are ready for release :)

Comment (stubbedev) [reply views/templates/hook/search-top.tpl:26]: Yes sir, I shall do the needful

**Review by CasperKN — COMMENTED** [review]
Nice catch  :)

**Pull request merged**